apps/ewallet_db/lib/ewallet_db/types/virtual_struct.ex

```diff
@@ -0,0 +1,23 @@
+defmodule EWalletDB.Types.VirtualStruct do
```

Comment:
Ummm.. Can you explain a bit more why we need VirtualStruct? Strings won't work? :S

Reply:
Well, it's to pass the actual struct without worrying about its type or having to add a formatter I guess. What would you suggest?